Fixes to #include statements, needed on jammy:

  chrome/browser/devtools/protocol/pwa_handler.cc:7:10: fatal error: 'expected' file not found
  #include <expected>
           ^~~~~~~~~~

--- a/chrome/browser/devtools/protocol/pwa_handler.cc
+++ b/chrome/browser/devtools/protocol/pwa_handler.cc
@@ -4,8 +4,6 @@
 
 #include "chrome/browser/devtools/protocol/pwa_handler.h"
 
-#include <expected>
-
 #include "base/check.h"
 #include "base/functional/bind.h"
 #include "base/numerics/clamped_math.h"
